### PR TITLE
use quote where we can

### PR DESCRIPTION
**Pull request by bucko909**

### CODE REVIEW

File: src/capnp_compile.erl [hunk 1]
```diff
@@ -323,37 +323,37 @@ generate_decode_fun(Line, TypeId, SortedDataFields, SortedPtrFields, Groups, Sch
 	end,
 	Name = decoder_name(TypeId, Schema),
 	ast_function(
-		{'$uberpt_quote', Name},
+		quote(Name),
 		fun
-			({'$uberpt_quote', DataMatcher}, {'$uberpt_quote', PointerMatcher}, {'$uberpt_quote', MessageRefVar}) ->
+			(quote(DataMatcher), quote(PointerMatcher), quote(MessageRefVar)) ->
```
Comment (kvelicka) [inline src/capnp_compile.erl]: This is the thing that was breaking on 21?

**Comment by kvelicka** [comment]
I'm not sure what this MR aims to do - it basically undoes the `$uberpt_quote` patch and re-breaks the lib for Erlang 21.

File: src/capnp_compile.erl (hunk 1 above)
Comment (bucko909) [reply src/capnp_compile.erl]: Ooops, off by one error.

**Comment by bucko909** [comment]
It uses `quote()` where we *can*, rather than avoiding it everywhere.

**Pull request merged**